fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/AbstractSumoAmbassador.java

```diff
@@ -1239,7 +1239,7 @@ private void setExternalVehiclesToLatestPositions() {
                                 external.getKey(),
                                 latestVehicleData.getPosition().toCartesian(),
                                 latestVehicleData.getHeading(),
-                                VehicleSetMoveToXY.Mode.KEEP_ROUTE
+                                VehicleSetMoveToXY.Mode.EXACT_POSITION
```

Comment:
1. better use `SWITCH_ROUTE` instead, that should move the vehicle to the closest edge and it should not happen that the vehicle drives outside the network. (see https://sumo.dlr.de/docs/TraCI/Change_Vehicle_State.html#move_to_xy_0xb4) 
2. make this configurable in `sumo_config.json`